projects/components/src/gauge/gauge.component.ts

```diff
@@ -112,13 +113,13 @@ export class GaugeComponent implements OnChanges {
   }
 
   private buildRadius(boundingBox: ClientRect): number {
-    return Math.min(boundingBox.height, boundingBox.width) / 2;
+    return Math.min(boundingBox.height, boundingBox.width) - GaugeComponent.GAUGE_AXIS_PADDING;
   }
 
-  private buildOrigin(boundingBox: ClientRect, radius: number): Point {
+  private buildOrigin(boundingBox: ClientRect, _: number): Point {
     return {
       x: boundingBox.width / 2,
-      y: boundingBox.height / 2 + radius / 2
+      y: boundingBox.height - GaugeComponent.GAUGE_AXIS_PADDING
```

Comment:
In the case where our height is greater than our width, I believe this is is incorrect -
Take, for example a width of 300 and height of 600 - we calculate a radius of `300 - 30 = 270`. Our origin here would be at `(150, 570)` - instead, we'd likely want it centered which would be more like `(150, 450)` - the padding not being needed because it's already accounted for in the centering. I think something like `min(height - padding, height / 2 + min(height, width) / 2)` => `min(570, 450)`